files/zh-cn/mdn/writing_guidelines/index.md

```diff
@@ -49,17 +61,34 @@ MDN Web 文档是一个开源项目。下面的概述部分描述了有关我们
 
   - : 本节介绍了我们用于在 MDN Web 文档中提供一致的信息呈现的各种页面结构。包括：
 
+    - [语法部分](/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Syntax_sections)
+
+      - : MDN Web 文档中的参考页面的语法部分包含一个语法框，用于定义一个特性的确切语法。本文介绍了如何为参考文档编写语法框。
+
     - [代码示例](/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Code_examples)
 
       - : 在页面上包含示代码示例的方法很多，本节介绍了这些方法，并提供了不同语言的语法指南。
 
+    - [横幅和说明](/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Banners_and_notices)
+
+      - : 有时，在文章中需要添加特殊的说明。如果页面中包含不推荐使用的技术或不应在生产环境中使用的其它内容，则可能会包含这些特殊说明。本文涵盖了最常见的情况，以及应该如何处理它们。
+
+    - [规范表格](/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Specification_tables)
+
+      - : MDN Web 文档中的每一个参考页面都应该提供有关定义该 API 或技术的规范信息。本文演示了这些表格的外观，并说明了添加的方法。
+
+    - [兼容性表格](/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Compatibility_tables)
+
+      - : MDN Web 文档为我们的开放式的 web 文档提供了兼容性表格的标准格式。本文说明了为用于生成兼容性表格的数据库添加和维护数据，以及如何将这些表格集成到文章中。
+
     - [宏](/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Macros)
 
-      - : 宏是页面中用于生成内容的较为便捷的方式，例如侧边栏。本节列出了我们使用的宏以及它们的作用。
+      - : 宏是页面中用于生成内容（例如侧边栏）的较为便捷的方式。本节列出了我们使用的宏以及它们的作用。
+
+- [署名和版权许可协议](/zh-CN/docs/MDN/Writing_guidelines/Attrib_copyright_license)
+
+  - : 描述在其它地方使用 MDN Web 文档内容的政策、如何获得重新发布 MDN 上的内容的许可，以及链接到 MDN 上的内容的相关说明。
 
 - [如何标记一项技术](/zh-CN/docs/MDN/Writing_guidelines/Experimental_deprecated_obsolete)
 
   - : 本节涵盖了我们对过时、弃用和实验性术语的定义，并提供了关于何时从 MDN Web 文档中删除相应内容的指南。
```

Comment:
I make a [PR](https://github.com/mdn/content/pull/23624) in mdn/content. let's expect the result.